src/Controller/PresentationController.php

```diff
@@ -14,8 +15,13 @@ public function __construct(private readonly PresentationRenderer $presentationR
 
     public function presentationAction(): Response
     {
+        $slidesMarkup = '';
+        if ($this->document instanceof PageSnippet) {
+            $slidesMarkup = $this->presentationRenderer->renderPresentation($this->document);
+        }
```

Comment:
Maybe we should throw instead? A presentation doesn't really make sense without slides, does it?